docs/architecture/011-capi-elements-as-components.md

```diff
@@ -4,6 +4,10 @@
 
 CAPI provides a representation of content which is strongly typed and like components.
 
+Frontend uses this representation to render liveblogs.
+
+_Note: This document has been updated with the knowledge that the `model.liveblog` representation is a dotcom abstraction and not directly from CAPI._
```

Comment:
That’s interesting. So there’s no reason we need to stick with the model.liveblog type name? Maybe we can choose a consistent type name for all elements used by dotcom-rendering? BTW I think it would be easier if the changes to this file were made as a separate PR